From 36484b7d6f3cba5390125c8e2cdb4168695a3a01 Mon Sep 17 00:00:00 2001
From: "Barry Lind (SteeManMI)" <barrylind@yahoo.com>
Date: Tue, 27 Dec 2022 16:08:43 -0500
Subject: [PATCH] In researching a post in the forums, I found that the
 meson64-edge kernel config doesn't include rtl8822cs (which apparently some
 TV boxes use). This is enabled in most kernel configs:

linux-bcm2711-current.config:CONFIG_RTL8822CS=m
linux-bcm2711-edge.config:CONFIG_RTL8822CS=m
linux-imx6-current.config:CONFIG_RTL8822CS=m
linux-imx6-edge.config:CONFIG_RTL8822CS=m
linux-media-current.config:CONFIG_RTL8822CS=m
linux-media-edge.config:CONFIG_RTL8822CS=m
linux-media-legacy.config:# CONFIG_RTL8822CS is not set
linux-meson64-current.config:CONFIG_RTL8822CS=m
linux-meson64-edge.config:# CONFIG_RTL8822CS is not set
linux-mvebu64-current.config:CONFIG_RTL8822CS=m
linux-mvebu64-edge.config:CONFIG_RTL8822CS=m
linux-mvebu-current.config:CONFIG_RTL8822CS=m
linux-mvebu-edge.config:CONFIG_RTL8822CS=m
linux-odroidxu4-edge.config:CONFIG_RTL8822CS=m
linux-rk322x-current.config:CONFIG_RTL8822CS=m
linux-rk322x-edge.config:CONFIG_RTL8822CS=m
linux-rockchip64-current.config:CONFIG_RTL8822CS=m
linux-rockchip64-edge.config:CONFIG_RTL8822CS=m
linux-rockchip-current.config:CONFIG_RTL8822CS=m
linux-rockchip-edge.config:# CONFIG_RTL8822CS is not set
linux-rockchip-rk3588-legacy.config:# CONFIG_RTL8822CS is not set
linux-sunxi64-current.config:CONFIG_RTL8822CS=m
linux-sunxi64-edge.config:CONFIG_RTL8822CS=m
linux-sunxi64-legacy.config:CONFIG_RTL8822CS=m
linux-sunxi-current.config:CONFIG_RTL8822CS=m
linux-sunxi-edge.config:CONFIG_RTL8822CS=m
linux-sunxi-legacy.config:CONFIG_RTL8822CS=m
linux-uefi-arm64-current.config:CONFIG_RTL8822CS=m
linux-uefi-arm64-edge.config:# CONFIG_RTL8822CS is not set
linux-uefi-riscv64-current.config:# CONFIG_RTL8822CS is not set
linux-uefi-riscv64-edge.config:# CONFIG_RTL8822CS is not set
linux-uefi-x86-current.config:CONFIG_RTL8822CS=m
linux-uefi-x86-edge.config:CONFIG_RTL8822CS=m

This commit enables this for just the meson64-edge kernel as that resolves the
issue I was researching, but I suspect it probably should also be enabled for
rockchip-edge as well, but I don't have any such devices.

On branch rtl8822cs_meson64_edge
Changes to be committed:
	modified:   ../kernel/linux-meson64-edge.config
---
 config/kernel/linux-meson64-edge.config | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/config/kernel/linux-meson64-edge.config b/config/kernel/linux-meson64-edge.config
index d8c7caa4f..7175a238a 100644
--- a/config/kernel/linux-meson64-edge.config
+++ b/config/kernel/linux-meson64-edge.config
@@ -3595,7 +3595,7 @@ CONFIG_WLCORE_SPI=m
 CONFIG_WLCORE_SDIO=m
 CONFIG_WILINK_PLATFORM_DATA=y
 CONFIG_RTL8723DU=m
-# CONFIG_RTL8822CS is not set
+CONFIG_RTL8822CS=m
 CONFIG_RTL8822BU=m
 CONFIG_RTL8821CU=m
 CONFIG_88XXAU=m
-- 
2.34.1

